File: inspect_css.js
const fs = require('fs');
const content = fs.readFileSync('assets/css/style.css', 'utf8');
const lines = content.split('\n');
console.log(`Total lines: ${lines.length}`);
for (let i = 1855; i < 1865; i++) {
    if (i < lines.length) {
        console.log(`Line ${i + 1}: ${JSON.stringify(lines[i])}`);
    }
}

File: inspect_css.py
with open("assets/css/style.css", "r", encoding="utf-8") as f:
    lines = f.readlines()
    print(f"Total lines: {len(lines)}")
    for i in range(1855, 1865):
        if i < len(lines):
            print(f"Line {i+1}: {repr(lines[i])}")
